### PR TITLE
enable_user_defined_functions

### PR DESCRIPTION
**Pull request by zhaxelll**
after update image https://github.com/bitnami/bitnami-docker-cassandra/pull/63
we want use this.
######
we need to create functions, but we do not want to use alternative cassandra.yaml

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi,

Thanks for the PR. Could you please update the chart minor version? It is adding a non-breaking feature.

**Comment by zhaxelll** [comment]
Hi. 
I was updated Chart.yaml

File: bitnami/cassandra/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: cassandra
-version: 5.0.3
+version: 5.0.4
```
Comment (carrodher) [inline bitnami/cassandra/Chart.yaml]:
```suggestion
version: 5.1.0
```

Comment (zhaxelll) [reply bitnami/cassandra/Chart.yaml]: done

**Comment by carrodher** [comment]
Thanks!

**Pull request merged**